contrib/unix/installer/Getopt.pm

```diff
@@ -182,7 +182,7 @@ glpi-agent-linux-installer [options]
                                      - "snap": don't install but extract snap package
     --runnow                       run agent tasks on installation (false)
     --type=INSTALL_TYPE            select type of installation (typical)
-                                     - "typical" to only install inventory task
+                                     - "typical" to only install Computer Inventory and Remote Inventory tasks
```

Comment:
There's no need to use `computer` as this is implicit. You should also avoid first letter capitalization.
```suggestion
                                     - "typical" to only install inventory and remote inventory tasks
```

Reply:
My suggested edit was based on how the info is presented on the Agent detail screens within GLPI.

My personal confusion (which you thankfully cleared up in the Q&A discussion) was thinking that "inventory" covered all the inventory tasks, not just the "Computer Inventory" task. Hoping to help the next person avoid my mistake.

<img width="937" alt="Screenshot 2024-01-04 at 12 03 58" src="https://github.com/glpi-project/glpi-agent/assets/30990146/95fe6571-b118-4cdc-bc42-9f1642258788">

Reply:
Okay, I understand your choice. Then can you just remove the first letter capitalization ?
```suggestion
                                     - "typical" to only install computer inventory and remote inventory tasks
```

Reply:
That's fine by me. I was just matching the text from the web UI.